### PR TITLE
Daniel/minor fixes

### PR DESCRIPTION
**Pull request by alined908**
1. Hide username/password
Note:  Need to do on console, heroku config:set MAILER_EMAIL=example@example.com MAILER_PASSWORD=password for whoever deploys

2. Cuke test for ranking#edit

### CODE REVIEW

**Review by Aakup — APPROVED** [review]
Nice Job!

File: features/roster.feature [hunk 1]
```diff
@@ -65,13 +65,36 @@ Feature: Roster
 
   Scenario: successfully email roster
     When I press the roster bottom for project of id "1"
-    When I fill in "Email" with "testing1@berkeley.edu"
+    When I fill in "Email" with "daniellee0228@berkeley.edu"
```
Comment (Aakup) [inline features/roster.feature:68]: Every-time we run the cucumber tests you would receive an email, are you sure you want to live in this life style.

**Review by kev-chien — COMMENTED** [review]
Looks good, but how does it access the environment variable MAILER_EMAIL and password? It seems to work automatically but I'd just like to make sure how it works, since you've deleted the account and password from the repo. Or has my rails just remembered the variables from a previous execution?

**Review by kev-chien — APPROVED** [review]
Looks good, thanks for making it secure and adding tests!

**Comment by alined908** [comment]
oh in the git ignore theres a config/initialize file that is hidden and has the vars set

**Comment by kev-chien** [comment]
> oh in the git ignore theres a config/initialize file that is hidden and has the vars set

Can you email / slack this file to us? Thanks!

**Pull request merged**